lib/core/api/index.js

```diff
@@ -1,9 +1,11 @@
 exports.bucket = require('./bucket.js');
-exports.createFile = require('./create-file.js');
+exports.uploadData = require('./upload-data.js');
+exports.getData = require('./get-data.js');
 exports.getFile = require('./get-file.js');
+exports.createFile = require('./create-file.js');
 exports.createFileToken = require('./create-file-token.js');
 exports.getFilePointers = require('./get-file-pointers');
-exports.initStore = require('./init-store.js');
 exports.resolveFileFromPointers = require('./resolve-file-from-pointers.js');
-exports.getKeyPair = require('./get-keypair.js');
+exports.storeFileInBucket = require('./store-file-in-bucket.js');
```

Comment:
agreed, having `uploadData` and `storeFileInBucket` exposed in the api is redundant. Will move that internally.